packages/webapp-server/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: admin
+      MONGO_INITDB_ROOT_PASSWORD: password
+    volumes:
+      - podwatch-data:/data/db
+  app:
+    build: .
+    restart: always
+    depends_on:
+      - mongodb
+    environment:
+      MONGODB_URI: mongodb://admin:password@mongodb:27017
+    # env_file:
+    #   - .env
```

Comment:
Later when we do add a .env file to the project, these lines will need to be uncommented.